### DISCUSSION TITLE
Scrollable tbody in tables

### DISCUSSION THREAD

**Discussion by Gregory-Lange** [twbs/bootstrap#39658] [Q&A]
I know in past versions of bootstrap even when I put tables inside other items, cards, containers, etc., I was able to make the body scroll with only minor work.  However in bootstrap 5.3 I am unable to do so unless I really get some things changed but then as soon as I change those items tables become a nightmare to maintain per page.  Does anyone have a way of doing this with minimal impact?  I am about to just give up on Bootstrap tables and get the theme from it but make my own sense there seems to be an issue in the default tables.  Any help would be wonderful.

What I have done to get the body to scroll:

tbody class
display: table-caption;
overflow-y: auto;

then depending on @media height
height: XXXX;

everything else causes the table to not look correct or still doesn't allow for a scrolling body.

Thanks,
Greg

**Comment by julien-deramond** [comment]
Could you maybe create a reproducible use case with CodePen, StackBlitz, etc. that used to work before and not anymore so that the community can start digging from something?

**Reply by Gregory-Lange** [reply]
I can do that I will get one done as soon as possible.

**Comment by Vanek131** [comment]
ye

**Comment by patkon** [comment]
You can do this by using an additional div and the classes .sticky-top / .top-0

```
<div class="position-relative" style="height: 100px; overflow-x: auto;">
<table class="table table-striped">
<thead class="sticky-top top-0">
...
...
...

</table>
</div>

**Comment by shahzad233** [comment]
i can do that one done as soon as possible
